### ISSUE TITLE
Slider in Hours form is difficult to use

### ISSUE THREAD

**Issue by mscholtz** [washingtontrails/vms#101]
I find it pretty difficult using the sliders on the hours entry form to actually select the number I want.  For instance, w/ my mouse I found it next to impossible to select 6 hours.  I could get 6.1 or 5.9, but not 6.  

If this is the only way to assign a number to all participants, I think we should make the slider easier (easier to get to whole numbers, for example), or just use a text box.

Also: if I've already hand-entered a number for one of the participants, and then slide the slider, it overwrites what I put in there.  This is at least mildly unexpected, and easy to do my accident, especially on a touchscreen.  I think we should consider only changing the values that haven't been hand-entered already.

**Comment by davisagli** [comment]
The slider is a standard HTML range input. We just give it a min, max, and step size, and you get whatever slider the browser implements. I guess it could be made easier by increasing the step size from the current value of 0.1, or by making the column wider so you have better control. Or using a text box would be an easy switch.

The behavior of overriding all the values was what we discussed when we implemented the story. It can be changed with some effort, if desired.

**Comment by WaBirder57** [comment]
The other consideration that Darrell was factoring in was the use of this from a mobile device.  He thought that the slider would be easier to use in that scenario.

**Comment by alecpm** [comment]
Seems like 0.25 would be a more reasonable step choice.

**Comment by WaBirder57** [comment]
Note that this would .2 as we only have one decimal digit.  Thus 6.0, 6.2, 6.4...

**Comment by alecpm** [comment]
To me it seems a bit strange to parcel out an hour into 6 or 12 minute intervals, rather than 15 (or 10 or 20, but those are worse to decimal notate).

**Comment by WaBirder57** [comment]
Yes, we agree but didn't want to go to 2 decimal digits due to space restraints and inferred precision from 2 decimal digits.  Would we allow 6.87 hrs?  Informal "pool" of crew leaders helped us decided they could handle 1 decimal digit and the fact of 6min intervals.
